src/lib.rs

```diff
@@ -493,6 +493,31 @@ impl Context {
         }
     }
 
+    /// Unseal and return data from a Sealed Data Object
+    pub fn unseal(&mut self, item_handle: ESYS_TR) -> Result<TPM2B_SENSITIVE_DATA> {
+        let mut out_data = null_mut();
+
+        let ret = unsafe {
+            Esys_Unseal(
+                self.mut_context(),
+                item_handle,
+                self.sessions.0,
+                self.sessions.1,
+                self.sessions.2,
+                &mut out_data,
+            )
+        };
+        let ret = Error::from_tss_rc(ret);
+
+        if ret.is_success() {
+            let out_data = unsafe { MBox::from_raw(out_data) };
+            Ok(TPM2B_SENSITIVE_DATA::try_from(*out_data).unwrap())
```

Comment:
Instead of `TPM2B_SENSITIVE_DATA::try_from(*out_data).unwrap()`, simply doing `*out_data` also works here with the same result!